Training/interface/DataLoader_main.h

```diff
@@ -819,11 +840,14 @@ class DataLoader {
             fillGrid(Br::ele_deltaPhiEleClusterTrackAtCalo, tau.ele_deltaPhiEleClusterTrackAtCalo.at(idx));
             fillGrid(Br::ele_deltaPhiSuperClusterTrackAtVtx, tau.ele_deltaPhiSuperClusterTrackAtVtx.at(idx));
             fillGrid(Br::ele_deltaPhiSeedClusterTrackAtCalo, tau.ele_deltaPhiSeedClusterTrackAtCalo.at(idx));
-            fillGrid(Br::ele_mvaInput_earlyBrem, tau.ele_mvaInput_earlyBrem.at(idx));
-            fillGrid(Br::ele_mvaInput_lateBrem, tau.ele_mvaInput_lateBrem.at(idx));
-            fillGrid(Br::ele_mvaInput_sigmaEtaEta, tau.ele_mvaInput_sigmaEtaEta.at(idx));
-            fillGrid(Br::ele_mvaInput_hadEnergy, tau.ele_mvaInput_hadEnergy.at(idx));
-            fillGrid(Br::ele_mvaInput_deltaEta, tau.ele_mvaInput_deltaEta.at(idx));
+            const bool cc_mvaInput = tau.ele_mvaInput_sigmaEtaEta.at(idx) >= -1e38;
+            if(cc_mvaInput) {
```

Comment:
I think this if-check can potentially break backward compatibility with the previous DeepTau training. 

On the other hand, maybe there was no such values (< -1e38) in data for v2.5. Or, we might not need to evaluate models trained with the current version of DataLoader (without the if-check), so it's OK to have this change merged. 

@shedprog what do you think?